core/platform/app_identity/gae_app_identity_services.py

```diff
@@ -39,14 +39,14 @@ def get_application_id():
 
 def get_gcs_resource_bucket_name():
     """Returns the application's bucket name for GCS resources, which depends
-    on the application ID.
+    on the application ID in production mode, or default bucket name in
+    development mode.
 
     This needs to be in sync with deploy.py which adds the bucket name to
     constants.js
 
     Returns:
-        str or None. The bucket name for the application's GCS resources, in
-        production mode.
+        str or None. The bucket name for the application's GCS resources..
```

Comment:
Drop the extra period at the end.

Please also explain in what circumstances will this function return None? It seems to me like that will never happen.

Reply:
Yes, it won't be None as the way things have been implemented.

Reply:
Done.